AusTraits_tutorial.qmd

```diff
@@ -2,104 +2,113 @@
 
 ## Introduction
 
-With more than 1.8 million data records, AusTraits is Australia's [largest plant trait database](austraits_database.html#plant_database), created using the [`traits.build`] package(https://github.com/traitecoevo/traits.build)
+With more than 1.8 million data records, AusTraits is Australia's [largest plant trait database](austraits_database.html#plant_database), created using the [`{traits.build}` R package](https://github.com/traitecoevo/traits.build)
 
+This tutorial introduces:
 
-The following tutorial introduces:
+-   [the database structure](#database_structure)
 
--   [the database structure](#database_structure)  
--   
--   additional examples of analyses using the database  
--   
+-   [`{austraits}` R package functions](#austraits_functions)
 
+-   additional [examples of analyses](#sample_analyses) using the database
 
-To access the Github repositories for individual components, see: 
+To access the Github repositories for individual components, see:
 
 -   the database structure, [`traits.build`](https://github.com/traitecoevo/traits.build)
```

Comment:
Should we point to the package or the book?

Reply:
I've rewritten to first point to book, then below mention there are individual GitHub repos